### PR TITLE
Feature/release plz

### PR DESCRIPTION
**Pull request by TobTheRock**

### CODE REVIEW

**Comment by TobTheRock** [comment]
Before merging this we need to wait until [release-plz-action]( https://github.com/MarcoIeni/release-plz-action) is updated to the latest version, so that the fix for https://github.com/MarcoIeni/release-plz/issues/687 is included

**Comment by TobTheRock** [comment]
We need to change this on organization level: :see_no_evil: 
image

**Comment by TobTheRock** [comment]
Everything should be setup , we can merge this now

**Review by hoodie — APPROVED** [review]

**Pull request merged**
